.github/workflows/ci.yml

```diff
@@ -3,20 +3,25 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  build:
+  test:
+    name: test w/ Python
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.10"]
     steps:
-      - name: Checkout source
-        uses: actions/checkout@v2
-
+      - uses: actions/checkout@v2
       - name: Install dependencies
-        run: |
-          pip install -r requirements-dev.txt
-          pip install flake8
-      
-      - name: flake8
-        run: flake8 .
-      
+        run: pip install -r requirements-dev.txt
       - name: Run tests
         run: pytest
+  
+  flake8-formatting:
+    name: code autoformatting with flake8
```

Comment:
This is misleading. We are not formatting or autoformatting anything. Linting would be the generic name for checking code styles, flake8 is the tool fot checking them.

Reply:
Great, thanks for the explanation :)